src/client/ui/menus/ToolBar.js

```diff
@@ -150,9 +159,25 @@ export default class ToolBar extends Component {
 
   render() {
     const { toolButtons, spaceToggle, snapToggle } = this.state;
+    const { updateAvailable, updateRequired, latestVersion, downloadUrl } = this.props.editor.updateInfo;
+    const showUpdateNotification = updateAvailable && !updateRequired && !this.state.updateNotificationDismissed;
     return (
       <div className={styles.toolbar}>
         <div className={styles.logo}>spoke</div>
+        {showUpdateNotification && (
+          <div className={styles.notification}>
+            <div className={styles.message}>
+              {`Spoke ${latestVersion} is available. `}
+              <Button className={styles.download} href={downloadUrl}>
```

Comment:
should we target blank this, so they don't lose work?

Reply:
Firefox just starts the download. Will check Chrome.

Reply:
Chrome does the same. The response from the URL that github provides includes a content disposition header, so browsers just download the release.